engine/source/output/h3d/h3d_results/h3d_shell_scalar_1.F

```diff
@@ -157,7 +157,7 @@ SUBROUTINE H3D_SHELL_SCALAR_1(
 
           DO I=1,NEL
             VALUE(I) = ZERO
-            IS_WRITTEN_VALUE(I) = 0
+            IS_WRITTEN_VALUE(I) = 1
```

Comment:
@SaidMamouri 

You can not do this initialization for all the outputs , this will lead to fill every values with 0. , for all the outputs .
you need only to put IS_WRITTEN_VALUE to 1 in the cae/HD/SHELL/USER & TYPE51/52 , for the existing user variables of the material law that is used .